types/picturefill/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for picturefill 3.0
+// Project: https://scottjehl.github.io/picturefill/
+// Definitions by: Alexander Azarov <https://github.com/alaz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Picturefill {
+  type ElementNullable = Element | null;
+
+  interface EvaluateArg {
+    reevaluate?: boolean;
+    elements: NodeList | ElementNullable[];
+  }
+}
+
+declare function picturefill(arg?: Picturefill.EvaluateArg): void;
+
+declare module 'picturefill' {
```

Comment:
I think you can use [the module-function template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html) instead, turning these 3 lines into

```ts
export = picturefill;
export as namespace picturefill;
```

The first lets you use `import picturefill = require('picturefill')` for node and the latter gives you global access as in the tests.

Reply:
Thank you! Works perfectly.